packages/main/src/themes/SegmentedButtonItem.css

```diff
@@ -0,0 +1,161 @@
+	background: var(--sapButton_Hover_Background);
+	border: 1px solid var(--sapButton_Hover_BorderColor);
+	color: var(--sapButton_Hover_TextColor);
+}
+
+.ui5-segmented-button-item-icon {
+	color: inherit;
+	flex-shrink: 0;
+}
+
+:host([icon-end]) .ui5-segmented-button-item-root {
+	flex-direction: row-reverse;
+}
+
+:host([icon-end]) .ui5-segmented-button-item-icon {
+	margin-inline-start: var(--_ui5_button_base_icon_margin);
+}
+
+:host([icon-only]) .ui5-segmented-button-item-root {
+	min-width: auto;
+	padding: 0;
+}
+
+:host([icon-only]) .ui5-segmented-button-item-text {
+	display: none;
+}
+
+.ui5-segmented-button-item-text {
+	outline: none;
+	position: relative;
+	white-space: inherit;
+	overflow: inherit;
+	text-overflow: inherit;
+}
+
+:host([has-icon]:not([icon-end])) .ui5-segmented-button-item-text {
+	margin-inline-start: var(--_ui5_button_base_icon_margin);
+}
+
+:host([has-icon][icon-end]) .ui5-segmented-button-item-text {
+	margin-inline-start: 0;
+}
+
+:host([disabled]) {
+	opacity: var(--sapContent_DisabledOpacity);
+	pointer-events: unset;
+	cursor: default;
+}
+
+:host([has-icon]:not([icon-only])) .ui5-segmented-button-item-text{
+	/* calculating the minimum width by removing the icon size  */
+	min-width: calc(var(--_ui5_button_base_min_width) - var(--_ui5_button_base_icon_margin) - 1rem);
+}
+:host([disabled]:active) {
+	pointer-events: none;
+}
+
+:host([desktop]:not([active])) .ui5-segmented-button-item-root:focus-within:after,
+:host(:not([active])) .ui5-segmented-button-item-root:focus-visible:after,
+:host([desktop][active]) .ui5-segmented-button-item-root:focus-within:before,
+:host([active]) .ui5-segmented-button-item-root:focus-visible:before {
+	content: "";
+	position: absolute;
+	box-sizing: border-box;
+	left: 0.0625rem;
+	top: 0.0625rem;
+	right: 0.0625rem;
+	bottom: 0.0625rem;
+	border: var(--_ui5_button_focused_border);
+	border-radius: var(--_ui5_button_focused_border_radius);
+}
+
+:host([desktop][active]) .ui5-segmented-button-item-root:focus-within:before,
+:host([active]) .ui5-segmented-button-item-root:focus-visible:before {
+	border-color: var(--_ui5_button_pressed_focused_border_color);
+}
+
+.ui5-segmented-button-item-root::-moz-focus-inner {
+	border: 0;
+}
+
+bdi {
+	display: block;
+	white-space: inherit;
+	overflow: inherit;
+	text-overflow: inherit;
+}
+
+:host([active][desktop]) .ui5-segmented-button-item-root:focus-within:after,
+:host([active]) .ui5-segmented-button-item-root:focus-visible:after,
+:host([pressed][desktop]) .ui5-segmented-button-item-root:focus-within:after,
+:host([pressed]) .ui5-segmented-button-item-root:focus-visible:after {
+	border-color: var(--_ui5_button_pressed_focused_border_color);
+	outline: none;
+}
+
+:host([desktop]:not(:last-child)) .ui5-segmented-button-item-root:focus-within:after,
+:host(:not(:last-child)) .ui5-segmented-button-item-root:focus-visible:after  {
+	border-top-right-radius: var(--_ui5_button_focused_inner_border_radius);
```

Comment:
We could use `border-start-end-radius` (for the top right) and `border-start-end-radius` (for bottom right radius') CSS Props for rtl responsiveness.